Added codes from examining "Tool Support"

kawilliams opened the pull request.

kawilliams committed:
Added codes from examining "Tool Support"

alex-r-bigelow reviewed (approved):
This is really interesting—wondering if there are two codes in that last point? (people need human help; people need more data collection)

kawilliams committed:
Split "adding info"

@alex-r-bigelow agreed, these should be separate. I originally tagged all of these responses as "ask_expert" (I'll show you my annotations later) but noticed the disconnect as I was doing it.

kawilliams commented:
@alex-r-bigelow agreed, these should be separate. I originally tagged all of these responses as "ask_expert" (I'll show you my annotations later) but noticed the disconnect as I was doing it.

alex-r-bigelow committed:
Add results from last week's meeting (forgot to commit this)
